src/CmdCallback.cpp

```diff
@@ -47,3 +47,34 @@ bool CmdCallbackObject::processCmd(CmdParser *cmdParser)
 
     return false;
 }
+
+void CmdCallbackObject::updateCmdProcessing(CmdParser *      cmdParser,
+                                            CmdBufferObject *cmdBuffer,
+                                            Stream * serial)
+{
+    // read data and check if command was entered
+    if (cmdBuffer->readSerialChar(serial)) {
+        // parse command line
+        if (cmdParser->parseCmd(cmdBuffer) != CMDPARSER_ERROR) {
+            // search command in store and call function
+            if (this->processCmd(cmdParser)) {
+                // FIXME: handling cmd not found
```

Comment:
They return True if the command is found, I'm not sure what the FIXME mean there. Maybe clear buffer only if a command will be found?

Reply:
Hi, the if-condition with the FIXME is a copy&paste from your code in function 'loopCmdProcessing' line 22. My understanding is that the code currently doen't implement any specific handling for unknown command. The if-condition is essentially useless until some code is added instead of the FIXME.

I ran the clang format on the changed files.

Reply:
You are right. Can you remove the useless code please and I merge the PR. Thanks. I will trigger the azure pipelines CI for this now

Reply:
I have removed the useless if-clause and inserted a comment instead.